src/main/java/chess/controller/ChessController.java

```diff
@@ -0,0 +1,81 @@
+package chess.controller;
+
+import chess.dto.ResponseDto;
+import chess.dto.ScoreDto;
+import org.eclipse.jetty.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import chess.model.room.Room;
+import chess.service.ChessService;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+public class ChessController {
+
+    private final ChessService chessService;
+
+    public ChessController(ChessService chessService) {
+        this.chessService = chessService;
+    }
+
+    @GetMapping("/")
+    public String home(Model model) {
+        model.addAttribute("rooms", chessService.getRooms());
+        return "home";
+    }
+
+    @PostMapping("/room")
+    public String room(@RequestBody String messageBody) {
+        final List<String> createRoomInput = Arrays.stream(messageBody.strip().split("\n"))
+                .map(s -> s.split("=")[1])
+                .collect(Collectors.toList());
+
+        Room room = chessService.init(
+                createRoomInput.get(0),
+                createRoomInput.get(1),
+                createRoomInput.get(2));
+
+        return "redirect:/room/" + room.getId();
+    }
+
+    @GetMapping("/room/{roomId}")
+    public String room(Model model, @PathVariable(value = "roomId") String roomId) {
```

Comment:
```suggestion
    @GetMapping("/room/{roomId}")
    public String room(Model model, @PathVariable String roomId) {
```
변수가 동일할 경우 value옵션을 생략할 수 있어요!